backend/infrastructures/apps/load/views/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class ExtractDataSerializer(serializers.Serializer):
```

Comment:
This should NOT be added in load app.
We need to have separate Django app, called `extract`